scripts/getData.py

```diff
@@ -1,135 +1,102 @@
-#!/usr/bin/env python
-
-from urllib2 import urlopen
-import string, os, time, json, re, unicodedata, HTMLParser
+#!/usr/bin/env python3
+import string, os, time, json, re, unicodedata, html.parser, requests
 
 # Bon appetit cafe hours api url
-url 	   = "http://legacy.cafebonappetit.com/api/2/cafes?cafe="
+url = "http://legacy.cafebonappetit.com/api/2/cafes"
 # How many cafeterias you want to parse (in order)
-totalCafes = 1374
+totalCafes = 10
 # What our file should be named
 fileName = "data.json"
 
 
 # Our constructed JSON data
 responseData = []
```

Comment:
Why is this a global? Just pass it into appendData and return the edited value from there.

Reply:
It is global because it's the one thing that is modified in this script by someone using it.

Reply:
responseData

Reply:
I'm going to opt to leave this as a global. It is the only thing in the script that gets tweaked.

Reply:
Oh. You meant `responseData`. I'll make that change.